content/swagger/resources/measurement_families/routes/measurement_families.yaml

```diff
@@ -163,3 +163,46 @@ get:
       $ref: "#/responses/403Error"
     406:
       $ref: "#/responses/406Error"
+patch:
+  summary: Update/create several measurement families
+  operationId: "patch_measurement_families"
+  tags:
+    - Measurement family
+  x-versions:
+    - "Serenity"
+  description: This endpoint allows you to update and/or create several measurement families at once.
+  x-body-by-line: Contains several lines, each line is a measurement family in JSON standard format
+  parameters:
+    - name: body
+      in: body
+      schema:
+        $ref: "#/definitions/MeasurementFamily"
+        x-examples:
+          x-example-1:
+            '{"code":"AREA","labels":{"de_DE": "Fläche"}}'
+          x-example-2:
+            '{"code":"AREA","standard_unit_code":"METER"}'
+          x-example-3:
+            '{"code":"AREA","unit_code":"SQUARE_CENTIMETER"}'
```

Comment:
```suggestion
            '{"code":"AREA","standard_unit_code":"SQUARE_CENTIMETER"}'
```

Reply:
Also, can you use a different code? It makes more sense to create/update different measurement families.